.travis.linux.trusty.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+URL=ftp://ftp.simplesystems.org/pub/libpng/png/src/libpng16/libpng-1.6.23.tar.gz
+FILE=libpng-1.6.23.tar.gz
```

Comment:
A submodule is pretty similar to what you're doing here, it's just avoiding a different build system. I think that's nicer especially for OS X users since they're unlikely to have libpng installed into the system. But if you check `FindLibPNG` first, then it can still use the system one if available...